lib/components/more_details_view.dart

```diff
@@ -0,0 +1,299 @@
+import 'package:flutter/material.dart';
+import 'package:project_eureka_flutter/components/eureka_image_viewer.dart';
+import 'package:project_eureka_flutter/models/more_details_model.dart';
+import 'package:project_eureka_flutter/models/user_answer_model.dart';
+import 'package:project_eureka_flutter/screens/home_page.dart';
+import 'package:project_eureka_flutter/services/close_question_service.dart';
+import 'package:timeago/timeago.dart' as timeago;
+
+class MoreDetailsView extends StatefulWidget {
+  final MoreDetailModel moreDetailModel;
+  final bool isAnswer;
+  final UserAnswerModel userAnswerModel;
+  final bool isCurrUser;
+
+  MoreDetailsView({
+    this.moreDetailModel,
+    @required this.isAnswer,
+    this.userAnswerModel,
+    @required this.isCurrUser,
+  });
+  @override
+  _MoreDetailsViewState createState() => _MoreDetailsViewState();
+}
+
+class _MoreDetailsViewState extends State<MoreDetailsView> {
+  Row _profileNameAndIcon() {
+    final DateTime dateTime = DateTime.parse(widget.isAnswer
+            ? widget.userAnswerModel.answer.answerDate
+            : widget.moreDetailModel.question.questionDate)
+        .subtract(Duration(hours: 7));
+
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.start,
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Padding(
+          padding: const EdgeInsets.fromLTRB(0.0, 10.0, 15.0, 0.0),
+          child: CircleAvatar(
+            radius: widget.isAnswer ? 20.0 : 40.0,
+            backgroundImage: (widget.isAnswer
+                    ? widget.userAnswerModel.user.pictureUrl == ''
+                    : widget.moreDetailModel.user.pictureUrl == '')
+                ? AssetImage('assets/images/profile_default_image.png')
+                : NetworkImage(
+                    widget.isAnswer
+                        ? widget.userAnswerModel.user.pictureUrl
+                        : widget.moreDetailModel.user.pictureUrl,
+                  ),
+          ),
+        ),
+        Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            SizedBox(height: 15.0),
+            Row(
+              children: [
+                Text(
+                  widget.isAnswer
+                      ? '${widget.userAnswerModel.user.firstName} ${widget.userAnswerModel.user.lastName}'
+                      : '${widget.moreDetailModel.user.firstName} ${widget.moreDetailModel.user.lastName}',
+                  style: TextStyle(
+                      fontSize: widget.isAnswer ? 11.0 : 15.0,
```

Comment:
I think the font sizes are too small to be readable maybe try increasing the font size by 2px in each one?